src/subscription_manager/cli.py

```diff
@@ -190,28 +192,19 @@ def main(self):
             print(error)
 
 
-def system_exit(code, msgs=None):
+def system_exit(code: int, msg: Union[str, Exception, None] = None) -> None:
     """
-    Exit with a code and optional message(s). Saved a few lines of code.
+    Exits the process with an exit code and optional message(s).
+
+    :param code: A unix-style system exit code.
+    :param msgs: A system exit message, a single exception, or none. This parameter defaults to None.
```

Comment:
typo, s/msgs/msg/

Reply:
oops, nice catch! Thanks, should be fixed now.